Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:carbon-alpine
+FROM node:10.3-alpine
```

Comment:
why not `node:10-alpine` , `node:chakracore` or `node:latest` ?

Reply:
No preference on my side really, just used what is used in verdaccio master

Maybe @juanpicado can shared some light :-)

Reply:
`node:10-alpine` is the latest 10.x version, so please take this.

Reply:
👍